xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/accounts/AccountCurrenciesResult.java

```diff
@@ -0,0 +1,65 @@
+package org.xrpl.xrpl4j.model.client.accounts;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.immutables.value.Value;
+import org.xrpl.xrpl4j.model.client.XrplResult;
+import org.xrpl.xrpl4j.model.client.common.LedgerIndex;
+import org.xrpl.xrpl4j.model.transactions.Hash256;
+
+import java.util.List;
+
+/**
+ * The result of an account_currencies rippled call.
+ */
+@Value.Immutable
+@JsonSerialize(as = ImmutableAccountCurrenciesResult.class)
+@JsonDeserialize(as = ImmutableAccountCurrenciesResult.class)
+public interface AccountCurrenciesResult extends XrplResult {
+
+    static ImmutableAccountCurrenciesResult.Builder builder() {
+        return ImmutableAccountCurrenciesResult.builder();
+    }
+
+    /**
+     * The identifying Hash of the ledger version used to generate this response.
+     *
+     * @return A {@link Hash256} containing the ledger hash.
+     */
+    @JsonProperty("ledger_hash")
+    Hash256 ledgerHash();
```

Comment:
I think this may need to be an `Optional<Hash256>`. The docs say "May Be Omitted" and I think all the other result objects with a `ledgerHash` are typed as `Optional<Hash256>`